modules/iam-eks-role/data.tf

```diff
@@ -0,0 +1,8 @@
+data "aws_caller_identity" "current" {}
+
+data "aws_partition" "current" {}
+
+data "aws_eks_cluster" "main" {
+  for_each = var.cluster_service_accounts
```

Comment:
Add newline after meta-arguments like `count` and `for_each`.

Reply:
Done